configs/faster_rcnn/faster-rcnn_r50-caffe_fpn_ms-90k_coco.py

```diff
@@ -1,15 +1,23 @@
 _base_ = 'faster-rcnn_r50-caffe_fpn_ms-1x_coco.py'
```

Comment:
replace with faster-rcnn_r50-caffe_fpn + common/ms-90k?

Reply:
RandomChoiceResize parameter is not the same.

Reply:
ok